pkg/burner/create.go

```diff
@@ -66,15 +70,13 @@ func (ex *JobExecutor) setupCreateJob(mapper meta.RESTMapper) {
 		}
 		_, gvk := yamlToUnstructured(o.ObjectTemplate, cleanTemplate, uns)
 		mapping, err := mapper.RESTMapping(gvk.GroupKind())
-		if err != nil {
-			log.Fatal(err)
-		}
-		obj := &object{
-			gvr:        mapping.Resource,
-			objectSpec: t,
-			Object:     o,
-			namespace:  uns.GetNamespace(),
-			namespaced: mapping.Scope.Name() == meta.RESTScopeNameNamespace,
+		obj.namespace = uns.GetNamespace()
```

Comment:
```suggestion
        obj := &object{
            objectSpec: t,
            Object: o,
		    namespace = uns.GetNamespace()
        }
```
And remove the creation of `obj` before